samples/snippets/src/main/java/com/example/mediatranslation/Translate.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.mediatranslation;
+
+public class Translate {
```

Comment:
What's the purpose of this file?

Reply:
In order to make cmd line instructions work [README].
I could not find other way to trigger `mvn exec:java -DTranslate -Dexec.args="translate_from_file path/to/audio.raw"`

Reply:
We don't want CLI anymore.

Reply:
ok, will delete it